cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/ConfigManager.kt

```diff
@@ -78,18 +83,12 @@ class ConfigManager : AbstractVerticle() {
     private fun attachEndpoint(config: JsonObject) {
         val endpointId = config.getString(CogboardConstants.PROP_ENDPOINT)
         endpointId?.let {
-            config.put(CogboardConstants.PROP_ENDPOINT, endpoints
-                    .stream()
-                    .map { it as JsonObject }
-                    .filter {
-                        endpointId == it.getString("id")
-                    }.findFirst()
-                    .orElse(JsonObject()))
+            config.put(CogboardConstants.PROP_ENDPOINT, endpoint.readById(endpointId))}
```

Comment:
I would like to see it like this:
`config.put(CogboardConstants.PROP_ENDPOINT, Endpoint.from(config(), endpointId))`